aten/src/ATen/native/cuda/ReflectionPad.cu

```diff
@@ -240,13 +240,14 @@ __global__ void reflection_pad3d_out_kernel(
 
 template <typename scalar_t>
 __global__ void reflection_pad3d_backward_out_kernel(
+    PackedTensorAccessor64<const scalar_t, 5> grad_input_const,
     PackedTensorAccessor64<scalar_t, 5> grad_input,
```

Comment:
?? both?

Reply:
Right, this isn't ideal. The only reason I did this was because I had to change `parallel_reflection_pad3d`'s `input` arg to be a const accessor, and I wasn't sure of the best way to convert a `PackedTensorAccessor64<scalar_t,...>` into a `PackedTensorAccessor64<const scalar_t,...>`. I guess adding a method `PackedTensorAccessor64::to_const()` might be the way to go

Reply:
I'll fix that in a follow up

Reply:
Oh right, I remember now that I tried adding `GenericPackedTensorAccessor::to_const()`, but the issue was that the constructor for `GenericPackedTensorAccessor` is a host-only function, so it's not possible to create a new one from within `reflection_pad3d_backward_out_kernel`, since it's a device function. So I don't know of any way to convert a `PackedTensorAccessor64<scalar_t,...>` to `PackedTensorAccessor64<const scalar_t,...>` in a device function. As far as I know, it's not possible...